### PR TITLE
enable redis locally

### PR DESCRIPTION
**Pull request by samandmoore**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,16 @@ Casual friendly video chat!
...
+`brew install postgresql`.
+
+### Redis
+
+Make sure you have redis installed and running. You can install via
+`brew install redis`.
+
 ### Foreman
 
 Foreman makes it simpler to run the app, worker, clock, and ewbpack dev server processes together.
```
Comment (kfinn) [inline README.md:19]: did I really commit "`ewbpack`" in here? yuck

**Comment by kfinn** [comment]
:shipit:

File: README.md (hunk 1 above)
Comment (samandmoore) [reply README.md:19]: lol

Comment (samandmoore) [reply README.md]: fixed

File: README.md [hunk 2]
```diff
@@ -4,9 +4,19 @@ Casual friendly video chat!
...
+
+### Redis
+
+Make sure you have redis installed and running. You can install via
+`brew install redis`.
+
 ### Foreman
 
-Foreman makes it simpler to run the app, worker, clock, and ewbpack dev server processes together.
+Foreman makes it simpler to run the app, worker, clock, and webpack dev server processes together.
```
Comment (kfinn) [inline README.md:19]: ✨

**Pull request merged**